diff -Naur mixxx-1.7.0/src/soundsourceffmpeg.h mixxx-1.7.0.tpg/src/soundsourceffmpeg.h
--- mixxx-1.7.0/src/soundsourceffmpeg.h	2009-08-06 07:11:36.000000000 +0000
+++ mixxx-1.7.0.tpg/src/soundsourceffmpeg.h	2009-08-30 19:51:47.000000000 +0000
@@ -20,8 +20,8 @@
 
 #include <qstring.h>
 #include "soundsource.h"
-#include <ffmpeg/avcodec.h>
-#include <ffmpeg/avformat.h>
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
 
 class TrackInfoObject;
 
